src/Utility/utilityFunctions.js

```diff
@@ -0,0 +1,16 @@
+export const getSortedVideos = (videos, sortByLatest) => {
+    if (sortByLatest === true) {
+        return [...videos].sort((a, b) => {
+            return new Date(b['uploaded']) - new Date(a['uploaded'])
+        })
+    }
```

Comment:
You can write this in this way also
`
if (sortByLatest === true) {
        return [...videos].sort((a, b) => new Date(b['uploaded']) - new Date(a['uploaded']))
    }
`